editions/2015-02.md

```diff
@@ -0,0 +1,61 @@
+
+For the past 18 months [Micaela Ash](https://twitter.com/M_M_Norah) has done an amazing job building up the division summaries and policies on our latest project [They Vote For You](http://www.theyvoteforyou.org.au/)—it is a vast new resource of how our laws are made. Micaela has now moved on to new, exciting work, so she left us some parting instructions on how we can keep the resource in shape. Surely there are people on this list who could take a few minutes to summarise a division once every few weeks?
+
+### [Percs: searchable pecuniary interest disclosures](http://percs.otherchirps.net/?query=development)
+
+Following January’s #nswvotes meetup, [Chris Nilsson]() whipped up a way to search the text of the previous very inaccessible 2013-2014 NSW Legislative Assembly Pecuniary Interest Disclosures. A huge step forward in the accessibility of this information, and there is more in the works—see [the OAF Meetup’s discussion board](http://www.meetup.com/OpenAustralia-Foundation/messages/boards/thread/48640454).
+
+
+### [Cuttlefish is now an official Poplus component](http://poplus.org/posts/cuttlefish/)
+
+Cuttlefish is an easy-to-set-up transactional email server developed by OpenAustralia Foundation Founder [Matthew Landauer](https://twitter.com/matthewlandauer). Last year the OpenAustralia Foundation was awarded a Poplus mini grant to create [a hosted version of Cuttlefish](https://cuttlefish.oaf.org.au/) which is freely available to the Poplus and international civic tech communities. Read Matthew’s post, *[Email is your secret weapon](http://poplus.org/posts/email-is-your-secret-weapon/)*, on the Poplus blog to find out more. Now Cuttlefish has been accepted as an official component, more people will find out about and use it.
+
+[Poplus](http://poplus.org/) is a global federation for civic tech, developing components that can support projects everywhere. Join the [Poplus maillist](http://poplus.org/get-involved/) for updates and discussion.
+
+
+### [Federal Government announces the Digital Transformation Office](https://www.pm.gov.au/media/2015-01-23/establishment-digital-transformation-office)
+
+The Government is establishing a “small team of developers, designers, researchers and content specialists working across government ... to make services simpler, clearer and faster for Australian families and businesses.” This is an initiative that the Australian civic tech community have been looking forward to since the establishment of the [Government Digital Services (GDS) office in the UK](https://www.gov.uk/). It is an important project for the civic tech community to get involved with and help make it as effective as possible. You can read more about the DTO at [the Communication Minister’s FAQ](http://www.malcolmturnbull.com.au/policy-faqs/faqs-the-digital-transformation-office). GDS have also written [a blog post](https://gds.blog.gov.uk/2015/01/26/simpler-clearer-australia/) about the project.
```

Comment:
'This is an initiative that the Australian civic tech community have been looking forward to since the establishment of the [Government Digital Services (GDS) office in the UK](https://www.gov.uk/). ' is longwinded, doesn't sounds like OAF, assumes that OAF speaks for a civic tech community.